### ISSUE TITLE
Natural gas density error

### ISSUE THREAD

**Issue by dalyw** [icra/ecam#495]
Hello,

I believe the FD factor for natural gas here is too high, by a factor of 1000. The value of 0.75 also doesn't match the source (GHG protocol spreadsheet, linked [here](https://ghgprotocol.org/sites/default/files/2023-05/Emission_Factors_from_Cross_Sector_Tools_March_2017%20%281%29.xlsx)) which says 0.7. 

<img width="638" alt="Screen Shot 2023-08-01 at 6 48 29 PM" src="https://github.com/icra/ecam/assets/139182397/61b4dac3-17d3-4bf8-aac3-765c35e1360c">

Thanks for the great tool and appreciate your help!

**Comment by holalluis** [comment]
That's right, 0.7 is kg/m3, thanks for spotting! it will be corrected soon
